js/main.js

```diff
@@ -67,4 +40,247 @@ const toggleMenu = ()=> {
+                <li class="popup__language__item">${language3}</li>
+                <li class="popup__language__item">${language4}</li>
+                <li class="popup__language__item">${framework}</li>
+            </ul>
+            <div class="about__btn__container">
+                <button class="popup__button">
+                    See live
+                    <i class="fa-solid fa-arrow-up-right-from-square"></i>
+                </button>
+                <button class="popup__button">
+                    See source
+                    <i class="fa-brands fa-github"></i>
+                </button>
+            </div>
+        </div>    
+    </div>
+    `
+    popupWindow.appendChild(article)
+
+    document.body.style.overflow = 'hidden'
+    
+    const closePopup = document.querySelector('.popup__xmark')
+    closePopup.addEventListener('click', ()=> {
+        workSection.classList.remove('blurEffect')
+        navbarSection.style.display = 'flex'
+        document.body.style.overflow = 'scroll'
+        popupWindow.removeChild(article)
+        workBtn.forEach((btn) => {
+            btn.classList.remove('press')
+        })
+    })
+}
+
+// WORK BUTTONS    
+workBtn.forEach((btn) => {
+    btn.addEventListener('click', (e) => {
+        btn.classList.toggle('press')
+        const ind = e.target.id
+        createPopup(ind)
+        workSection.classList.add('blurEffect')
+        navbarSection.style.display = 'none'
+    })
+})
+
+// CONTACT FORM
+const form = document.getElementById('contact__form')
+const email = document.getElementById('email')
+const emailError = document.querySelector('#email + span.emailError')
+const nameInpt = document.getElementById('name')
+const nameError = document.querySelector('#name + span.nameError')
+
+const showMailError = ()=> {
+    if (email.validity.valueMissing) {
+        emailError.textContent = 'Enter a valid Email please.'
+    } else if (email.validity.tooShort) {
+        emailError.textContent = `Email has a min of ${email.minLength} characters, you entered ${email.value.length}.`
+    }
+}
+
+const showNameError = ()=> {
+    if (nameInpt.validity.valueMissing) {
+        nameError.textContent = 'Enter a valid Name please.'
+    } else if (nameInpt.validity.tooShort) {
+        nameError.textContent = `Email has a min of ${nameInpt.minLength} characters.`
+    }
+}
+
+email.addEventListener('input', ()=> {
+    if (email.validity.valid) {
+        emailError.textContent = ''
+        emailError.className = 'emailError'
+    } else {
+        showMailError()
+    }
+})
+
+nameInpt.addEventListener('input', ()=> {
+    if (nameInpt.validity.valid) {
+        nameError.textContent = ''
+        nameError.className = 'nameError'
+    } else {
+        showNameError()
+    }
+})
+
+form.addEventListener('submit', (event)=> {
+    if (!email.validity.valid) {
+        showMailError()
+        event.preventDefault()
+    }
+})
```

Comment:
- [ ] At the moment, when I submit a form with the email in uppercase like "PROM@GMAIL.COM" it submits successfully. Please write a validation checking if the email is written in all lowercase. If it isn't, an error message should show up.